hub/api/dataset.py

```diff
@@ -261,24 +261,37 @@ def mode(self):
     def pytorch(
         self,
         transform: Optional[Callable] = None,
-        workers: int = 1,
+        num_workers: int = 1,
         tensors: Optional[List[str]] = None,
+        batch_size: Optional[int] = 1,
+        drop_last: Optional[bool] = False,
+        collate_fn: Optional[Callable] = None,
+        pin_memory: Optional[bool] = False,
```

Comment:
**kwargs? (if they specify `shuffle` you can raise an error)

Reply:
We only support like 4-5 out of 14 total args of dataloader so it's better to only support these ones.